tests/ParserTest.elm

```diff
@@ -543,6 +543,22 @@ expr =
                   , "'A'"
                   , Ok (Literal (Char 'A'))
                   )
+                , ( "double quote"
+                  , "'\"'"
+                  , Ok (Literal (Char '\"'))
+                  )
+                , ( "emoji"
+                  , "'😃'"
+                  , Ok (Literal (Char '😃'))
+                  )
+                , ( "escaped single quote"
+                  , "'\\\''"
```

Comment:
I wondered if `"""'\''"""` wouldn't be more readable, but ... probably not :upside_down_face: